web/client/themes/default/icons.less

```diff
@@ -581,152 +582,153 @@ structure .glyphicon-{iconName}-content
-.glyphicon-size-large-content { content: "\f21f"; }
-.glyphicon-size-medium-content { content: "\f220"; }
-.glyphicon-size-small-content { content: "\f221"; }
-.glyphicon-sort-by-alphabet-alt-content { content: "\f222"; }
-.glyphicon-sort-by-alphabet-content { content: "\f223"; }
-.glyphicon-sort-by-attributes-alt-content { content: "\f224"; }
-.glyphicon-sort-by-attributes-content { content: "\f225"; }
-.glyphicon-sort-content { content: "\f226"; }
-.glyphicon-star-empty-content { content: "\f227"; }
-.glyphicon-star-content { content: "\f228"; }
-.glyphicon-stats-content { content: "\f229"; }
-.glyphicon-step-backward-content { content: "\f22a"; }
-.glyphicon-step-forward-content { content: "\f22b"; }
-.glyphicon-stop-content { content: "\f22c"; }
-.glyphicon-story-banner-section-content { content: "\f22d"; }
-.glyphicon-story-carousel-section-content { content: "\f22e"; }
-.glyphicon-story-immersive-content-content { content: "\f22f"; }
-.glyphicon-story-immersive-section-content { content: "\f230"; }
-.glyphicon-story-media-section-content { content: "\f231"; }
-.glyphicon-story-paragraph-section-content { content: "\f232"; }
-.glyphicon-story-title-section-content { content: "\f233"; }
-.glyphicon-story-webpage-section-content { content: "\f234"; }
-.glyphicon-tag-content { content: "\f235"; }
-.glyphicon-tags-content { content: "\f236"; }
-.glyphicon-tasks-content { content: "\f237"; }
-.glyphicon-text-background-content { content: "\f238"; }
-.glyphicon-text-colour-content { content: "\f239"; }
-.glyphicon-text-height-content { content: "\f23a"; }
-.glyphicon-text-width-content { content: "\f23b"; }
-.glyphicon-th-large-content { content: "\f23c"; }
-.glyphicon-th-list-content { content: "\f23d"; }
-.glyphicon-th-content { content: "\f23e"; }
-.glyphicon-thumbs-down-content { content: "\f23f"; }
-.glyphicon-thumbs-up-content { content: "\f240"; }
-.glyphicon-time-current-content { content: "\f241"; }
-.glyphicon-time-offset-content { content: "\f242"; }
-.glyphicon-time-content { content: "\f243"; }
-.glyphicon-tint-content { content: "\f244"; }
-.glyphicon-transfer-content { content: "\f245"; }
-.glyphicon-trash-square-content { content: "\f246"; }
-.glyphicon-trash-content { content: "\f247"; }
-.glyphicon-unchecked-content { content: "\f248"; }
-.glyphicon-undo-content { content: "\f249"; }
-.glyphicon-unplug-content { content: "\f24a"; }
-.glyphicon-upload-content { content: "\f24b"; }
-.glyphicon-usd-content { content: "\f24c"; }
-.glyphicon-user-content { content: "\f24d"; }
-.glyphicon-vert-dashed-content { content: "\f24e"; }
-.glyphicon-warning-sign-content { content: "\f24f"; }
-.glyphicon-webpage-content { content: "\f250"; }
-.glyphicon-wrench-content { content: "\f251"; }
-.glyphicon-zoom-in-content { content: "\f252"; }
-.glyphicon-zoom-out-content { content: "\f253"; }
-.glyphicon-zoom-to-content { content: "\f254"; }
+.glyphicon-map-context-content { content: "\f1c0"; }
```

Comment:
Looks strange to me that all the codes has been re-mapped. @allyoucanmap 
Is it the normal behavior of the script? In the past I've seen only a line added.

Reply:
@alexander-fedorenko , generation looks ok. Anyway I can not see the new svg in the folder. It have to be added too, in order to be present in future generations of icons.

Reply:
Thank, added svg file to the tree.